Base/Python/slicer/util.py

```diff
@@ -1091,3 +1091,97 @@ def onClick(caller,event):
   up()
   interactor.SetShiftKey(0)
   interactor.SetControlKey(0)
+
+def downloadFile(url, targetFilePath):
+  """ Download ``url`` to local storage as ``targetFilePath``
+      Target file path needs to indicate the file name and extension as well
+  """
```

Comment:
No need to align or add extra space. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

```
"""Download ``url`` to local storage as ``targetFilePath``.
"""
```

Also I don't think this implementation require the extension to be specified in the target file.

Reply:
There can be many levels of this. Specify both file name and extension, only specify file name, only specify path. I don't think any is better than the other, but we need to choose. As one is already implemented, I think it makes sense to choose that one.